crates/superposition_types/src/database/models/others.rs

```diff
@@ -162,3 +162,21 @@ impl From<Map<String, Value>> for CustomHeaders {
         Self(value)
     }
 }
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+#[cfg_attr(
+    feature = "diesel_derives",
+    derive(Queryable, Selectable, Insertable, AsChangeset)
+)]
+#[cfg_attr(feature = "diesel_derives", diesel(check_for_backend(diesel::pg::Pg)))]
+#[cfg_attr(feature = "diesel_derives", diesel(primary_key(name)))]
+pub struct Variable {
+    pub name: String,
```

Comment:
we can do this, no ?
```suggestion
    pub name: VariableName,
```